gravitee-apim-distribution/pom.xml

```diff
@@ -105,10 +105,14 @@
         <gravitee-fetcher-http.version>1.12.0</gravitee-fetcher-http.version>
         <gravitee-repository-elasticsearch.version>3.11.0</gravitee-repository-elasticsearch.version>
         <!-- Gateway Only -->
+        <gravitee-notifier-email.version>1.3.2</gravitee-notifier-email.version>
+        <gravitee-notifier-slack.version>1.2.1</gravitee-notifier-slack.version>
+        <gravitee-notifier-webhook.version>1.1.0</gravitee-notifier-webhook.version>
         <gravitee-reporter-elasticsearch.version>3.11.0</gravitee-reporter-elasticsearch.version>
         <!--    Version of policy-ratelimit is also used for policy-quota, policy-spikearrest and gateway-services-ratelimit    -->
         <!--    <gravitee-gateway-services-ratelimit.version>1.15.0</gravitee-gateway-services-ratelimit.version>    -->
         <gravitee-reporter-file.version>2.5.0</gravitee-reporter-file.version>
+        <gravitee-reporter-kafka.version>1.3.0</gravitee-reporter-kafka.version>
```

Comment:
and this

Reply:
Actually, we could remove this property since kafka reporter is not embedded in the distribution.
I removed it since it is not maintained by us.